README.rst

```diff
@@ -10,9 +10,10 @@ Why Should I Use This?
 ----------------------
 
 The biggest reason to use twine is that it securely authenticates you to PyPI
-over HTTPS using a verified connection while ``python setup.py upload`` `uses
-HTTP and exposes your credentials <http://bugs.python.org/issue12226>`_.  This
-means anytime you use it you expose your username and password to being
+over HTTPS using a verified connection while ``python setup.py upload`` `only
+recently stopped using HTTP <http://bugs.python.org/issue12226>`_ in Python
+2.7.9+ and Python 3.2+. This means anytime you use ``python setup.py upload``
+with an older Python version, you expose your username and password to being
 sniffed. Twine uses only verified TLS to upload to PyPI protecting your
```

Comment:
We should probably also say "easily sniffed" because there's no verification of the connection and someone using a tool like `mitmproxy` could easily sniff the credentials over TLS that is not verified.